client/src/authenticated-app.tsx

```diff
@@ -0,0 +1,13 @@
+import { SideBar } from './components/sidebar'
+import React from 'react'
+
+export const AuthenticatedApp = () => {
```

Comment:
where's the type

Reply:
For the component?

Reply:
Yes

Reply:
Maybe declare type after I set the authentication state? I haven't set up authentication yet.

Reply:
React.FC<>